File: web/eRentACar/ModelVM/AuthenticationResultVM.cs
namespace eRentACar.ModelVM
{
    public class AuthenticationResultVM
    {
        public int UserId { get; internal set; }
        public string UserName { get; internal set; }
        public string PasswordSalt { get; internal set; }
        public string Phone { get; internal set; }
        public string FirstName { get; internal set; }
        public string LastName { get; internal set; }
        public string Email { get; internal set; }
        public string Token { get; internal set; }
    }
}

File: android studio/app/src/main/java/android/fit/ba/erentacar/data/AuthenticationResultVM.java
package android.fit.ba.erentacar.data;


import java.io.Serializable;

public class AuthenticationResultVM implements Serializable {
    public int UserId;
    public String FirstName;
    public String LastName;
    public String UserName;
    public String PasswordSalt;
    public String Phone;
    public String Email;
    public String Token;
}